api/service/v1alpha1/service.proto

```diff
@@ -659,11 +659,15 @@ message GetRoleResponse {
   }
 }
 
+message ListUserClaims {
+  repeated github.com.akuity.kargo.api.rbac.v1alpha1.UserClaim user_claims = 1;
+}
```

Comment:
Is this new message type because `oneof` and `repeated` didn't play nicely together?

Reply:
Yes, like the implementation below in this file. Would you like the comment `// Note: oneof and repeated do not work together` to be added?

Reply:
Whoops. Overlooked that.

Reply:
Ahh, so you mean we don't want to be doing it this way?

Reply:
Oh okay, no worries then :)

Reply:
I added the comment, just to be clear as why it exists.